ListTests/ArrayListTests.cs

```diff
@@ -1,8 +1,389 @@
 using NUnit.Framework;
+using System;
 
 namespace List.Tests
 {
     public class ArrayListTests
     {
+        [TestCase(100, new int[] { 1, 2, 3, 4, 5 }, new int[] { 1, 2, 3, 4, 5, 100 })]
+        [TestCase(-10, new int[] { 1, 2, 3, 4, 5 }, new int[] { 1, 2, 3, 4, 5, -10 })]
+        [TestCase(0, new int[] { 1, 2, 3, 4, 5 }, new int[] { 1, 2, 3, 4, 5, 0 })]
+        public void Add_WhenValidValuePassed_ShouldAddToListinLastPosition(int value, int[] actualArray, int[] expectedArray)
+        {
+            ArrayList<int> actual = new ArrayList<int>(actualArray);
+            ArrayList<int> expected = new ArrayList<int>(expectedArray);
+
+            actual.Add(value);
+
+            Assert.AreEqual(expected, actual);
+        }
+
+        [TestCase(100, new int[] { 1, 2, 3, 4, 5 }, new int[] { 100, 1, 2, 3, 4, 5 })]
+        [TestCase(-10, new int[] { 1, 2, 3, 4, 5 }, new int[] { -10, 1, 2, 3, 4, 5 })]
+        [TestCase(0, new int[] { 1, 2, 3, 4, 5 }, new int[] { 0, 1, 2, 3, 4, 5 })]
+        public void AddFirst_WhenValidValuePassed_ShouldAddToListFisrPosition(int value, int[] actualArray, int[] expectedArray)
+        {
+            ArrayList<int> actual = new ArrayList<int>(actualArray);
+            ArrayList<int> expected = new ArrayList<int>(expectedArray);
+
+            actual.AddFirst(value);
+
+            Assert.AreEqual(expected, actual);
+        }
+
+        [TestCase(100, 0, new int[] { 1, 2, 3, 4, 5 }, new int[] { 100, 1, 2, 3, 4, 5 })]
+        [TestCase(-10, 1, new int[] { 1, 2, 3, 4, 5 }, new int[] { 1, -10, 2, 3, 4, 5 })]
+        [TestCase(0, 3, new int[] { 1, 2, 3, 4, 5 }, new int[] { 1, 2, 3, 0, 4, 5 })]
```

Comment:
Add case to add in the end of current array

Reply:
Test passed.